lib/Performance/Probability.pm

```diff
@@ -0,0 +1,306 @@
+package Performance::Probability;
+
+use 5.010;
+use strict;
+use warnings;
+use Moo;
+
+use Date::Utility;
+use Math::BivariateCDF;
+use Math::Gauss::XS;
+
+our $VERSION = '0.01_1';
+
+=head1 NAME
+
+Performance::Probability - The performance probability is a likelihood measure of a client reaching his/her current P&L. 
+
+=head1 VERSION
+
+0.01_1
+
+=head1 SYNOPSYS
+
+=head1 DESCRIPTION
```

Comment:
Please, add an usage example here, i.e .with real values (number) and the expected output